src/select/__tests__/select.test.tsx

```diff
@@ -536,6 +536,29 @@ describe.each([false, true])('expandToViewport=%s', expandToViewport => {
     });
   });
 
+  describe('Inline Label', () => {
+    test('should render', () => {
+      const testLabel = 'Test label';
+      const { wrapper } = renderSelect({ inlineLabelText: testLabel });
+      const labelElement = wrapper.findByClassName(selectPartsStyles['inline-label']);
```

Comment:
this class should become a test-util instead (in https://github.com/cloudscape-design/components/blob/main/src/test-utils/dom/select/index.ts), so that customers can also test similar things

Reply:
Done!